resources/schema/spidacalc/results/item.schema

```diff
@@ -29,7 +29,12 @@
       "$ref": "../../general/measurable.schema"
     },
     "reaction": {
-      "$ref": "../../general/point_load.schema"
```

Comment:
That code looks like it is a triad: https://github.com/spidasoftware/schema/blob/master/resources/schema/general/triad.schema